schemas/hardwareSystem.schema.tpl.json

```diff
@@ -1,20 +1,11 @@
 {
     "_type": "https://openminds.ebrains.eu/computation/HardwareSystem",
-    "required": [
-        "name"
-    ],
+    "_extends": "/core/schemas/products/device.schema.tpl.json",
+    "required": [],
```

Comment:
@apdavison the version is not a requirement? are version identifiers typical for hardware systems (that go on top of the version specific model name already asked for in device?

@Peyman-N depending on @apdavison response the "required" part just needs to be deleted from this schema completely (if you don't add anything it's defined by Device)